pkg/kubelet/certificate/certificate_manager.go

```diff
@@ -197,6 +197,7 @@ func (m *manager) Start() {
 	// loop to allow bootstrap scenarios, where the certificate manager
 	// doesn't have a certificate at all yet.
 	if m.shouldRotate() {
+		glog.V(1).Infof("shouldRotate() is true, forcing immediate rotation")
```

Comment:
this only happens on start up and is fairly important to know about (def: of V(1) log level)